server/models/Order.js

```diff
@@ -1,7 +1,7 @@
 const { Schema, model, Types } = require("mongoose");
 
 const orderSchema = new Schema({
-  product: [
+  products: [
```

Comment:
Anytime you find something that needs to be fixed and is not part of your current task, you branch out from develop, create a fix, push and raise PR @TmkrRina. Revert this, go to develop branch, create a branch specifically for fixing this bug. and raise a PR